float32 left_x
float32 left_y
float32 right_x
float32 right_y

float32 LT
float32 RT

bool A
bool B
bool X
bool Y
bool LAS
bool RAS

